halo2_proofs/src/plonk/circuit.rs

```diff
@@ -493,8 +571,9 @@ impl<F: Field> Expression<F> {
         constant: &impl Fn(F) -> T,
         selector_column: &impl Fn(Selector) -> T,
         fixed_column: &impl Fn(usize, usize, Rotation) -> T,
-        advice_column: &impl Fn(usize, usize, Rotation) -> T,
+        advice_column: &impl Fn(usize, usize, Rotation, Phase) -> T,
         instance_column: &impl Fn(usize, usize, Rotation) -> T,
+        challenge: &impl Fn(usize, Phase) -> T,
```

Comment:
```suggestion
        challenge: &impl Fn(Challenge) -> T,
```

Reply:
Updated in [`902965c`](https://github.com/zcash/halo2/pull/593/commits/902965c385833ada61def929905c7431ff36e78b).